Lightyear/Position.idr

```diff
@@ -0,0 +1,75 @@
+-- ------------------------------------------------------------ [ Position.idr ]
+-- Module    : Lightyear.Position
+-- Description : Positioning in source files.
+--
+-- This code is distributed under the BSD 2-clause license.
+-- See the file LICENSE in the root directory for its full text.
+-- --------------------------------------------------------------------- [ EOH ]
+module Lightyear.Position
+
+import Data.String
+
+%default total
+%access export
+
+||| Generic represention of a position.
+public export
+record Position where
+  constructor MkPos
+  fname   : Maybe String
+  lineNo  : Nat
+  colNo   : Nat
+  charOff : Nat
```

Comment:
What's the purpose of `charOff`? Why is separate from `colNo`? It's not clear to me from the code.